Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -2688,11 +2709,24 @@ void TestController::decidePolicyForNotificationPermissionRequest(WKPageRef page
 void TestController::decidePolicyForNotificationPermissionRequest(WKPageRef, WKSecurityOriginRef origin, WKNotificationPermissionRequestRef request)
 {
     auto originName = originUserVisibleName(origin);
+
+    auto securityOriginString = adoptWK(WKSecurityOriginCopyToString(origin));
+    auto dictionary = adoptWK(m_webNotificationProvider.notificationPermissions());
```

Comment:
adoptWK() is wrong here.

Reply:
I changed the implementation to not call m_webNotificationProvider.notificationPermissions().